pkg/namespacestore/namespacestore.go

```diff
@@ -562,6 +563,10 @@ func RunCreateS3Compatible(cmd *cobra.Command, args []string) {
 		sigVer, _ := cmd.Flags().GetString("signature-version")
 		secretName, _ := cmd.Flags().GetString("secret-name")
 		mandatoryProperties := []string{"AWS_ACCESS_KEY_ID", "AWS_SECRET_ACCESS_KEY"}
+		u, _ := url.Parse(endpoint)
+		if u.Scheme == "http" {
+			sigVer = ""
```

Comment:
We should add here a printing that the signature-version v2 will be used.
We should also fail here if a signature-version is provided and it is v4